src/search/landmarks/landmark_cost_assignment.cc

```diff
@@ -77,17 +80,20 @@ double LandmarkUniformSharedCostAssignment::cost_sharing_h_value(
         }
     }
 
+    // TODO: Replace with Landmarks
```

Comment:
We still need to access the id of the landmark node in line 120, so this is not trivially possible. However, from the design we discussed, I think it should be possible to do this without needing to access the landmark node, hence the comment. We have to think about if the id should be part of the landmark or how else it is possible to access the status of a landmark, but I would do that once we implement the new design and would hence keep this is at it is for now.

Reply:
I updated the comment to make this clearer

Reply:
@FlorianPommerening is this still relevant? If yes, please make a note in the pull request of issue1023 here (https://github.com/aibasel/downward/pull/58); otherwise, please mark as resolved.

Reply:
I think your comment above cleared this up. let's leave it as a TODO for now and handle it once we implement the new design.